themes/bootstrap5/templates/RecordTab/overdriveholdings.phtml

```diff
@@ -44,16 +45,16 @@
         <?php if (($avail->data->availabilityType ?? 0) == 'AlwaysAvailable'): ?>
           <?=$this->transEsc('od_unlimited')?>
         <?php else: ?>
-          <?=$this->transEsc('od_copies_available', ['%%copies%%' => $avail->data->copiesAvailable . '/' . $avail->data->copiesOwned])?>
+          <?=$this->transEsc('od_copies_available', ['%%copies%%' => $copiesAvailable . '/' . ($avail->data->copiesOwned ?? 0)])?>
```

Comment:
@maccabeelevine, do you think we should change the else to `else if ($avail->data->copiesOwned ?? 0 > 0):` so we display nothing if no copies are owned, instead of the weird 0/0 thing?

Reply:
Based on @bpalme's report that OverDrive itself shows "0 of 0" it's probably best to leave this alone for consistency.